src/Monitor/Tracing/Zipkin.hs

```diff
@@ -87,14 +89,16 @@ data Settings = Settings
   -- ^ An optional HTTP manager to use for publishing spans on the Zipkin server.
   , settingsPublishPeriod :: !(Maybe NominalDiffTime)
   -- ^ If set to a positive value, traces will be flushed in the background every such period.
+  , settingsIgnoreBackgroundExceptions :: Bool
+  -- ^ If set to True, background flushes will ignore any `HttpException`s
   }
 
 -- | Creates empty 'Settings'. You will typically use this (or the 'IsString' instance) as starting
 -- point to only fill in the fields you care about:
 --
 -- > let settings = defaultSettings { settingsPort = Just 2222 }
 defaultSettings :: Settings
-defaultSettings = Settings Nothing Nothing Nothing Nothing Nothing
+defaultSettings = Settings Nothing Nothing Nothing Nothing Nothing False
```

Comment:
I'd follow already used convention - let's use `Nothing`  as a default and interpret is as `Just False` inside `new`

Reply:
But that's an extra (and confusing, I must say) layer that makes no sense, isn't it? If `Nothing` means `Just False`, then `Just True` means `Just True`. And that's what `Bool` does and means... This is clearly a boolean value. And yes, I understand, it breaks a nice line of `Nothing`s ;)

Reply:
The reasoning behind me wanting to user `Nothing` as a default is that if you even want to use a default value, you just stick `Nothing` in it, you don't have to look up what a default value actually is (`True`/`False`). But I'm not sure if it's worth the confusion of having the default value (`Nothing`) interpreted somewhere in the codebase... :thinking:

Reply:
eh, screw it, let's keep it simple